Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Ft232h/Driver/Ft232h.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Threading;
+
+namespace Meadow.Foundation.ICs.IOExpanders
+{
+    /// <summary>
+    /// Represents a DS3502 digital potentiometer
+    /// </summary>
+    public partial class Ft232h : IDisposable
+    {
+        private bool _isDisposed;
+        private static int _instanceCount = 0;
+
+        static Ft232h()
+        {
+            if (Interlocked.Increment(ref _instanceCount) == 1)
+            {
+                // only do this one time (no matter how many instances are created instances)
+                Native.Functions.Init_libMPSSE();
+            }
+        }
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!_isDisposed)
+            {
+                if (Interlocked.Decrement(ref _instanceCount) == 0)
+                {
+                    // last instance was disposed, clean house
+                    Native.Functions.Cleanup_libMPSSE();
+                }
+                _isDisposed = true;
+            }
+        }
+
+        public I2cChannel[] GetI2CChannels()
```

Comment:
Can this follow the `II2cController` contract and implement `CreateI2cBus()`? There is an overload that takes a `busNumber`, I would think that the `channel` concept here translates to that. 

Maybe upon instantiation or lazily, whichever, we can interrogate the chip and create the appropriate `ChannelInfo`s?